src/components/FilterPanel/FilterPanel.jsx

```diff
@@ -2,6 +2,7 @@ import { useFilter, useNote } from "../../contexts";
 import { bxIcons } from "../../data/icons";
 import { Filteritem } from "../FilterItem/Filteritem";
 import "./FilterPanel.css";
+import "./FilterPanelMobile.css";
```

Comment:
css file name should be smallcase!

Reply:
I am following same naming convention for jsx and css files.